sklearn/utils/multiclass.py

```diff
@@ -234,6 +234,10 @@ def type_of_target(y):
         raise ValueError('Expected array-like (array or non-string sequence), '
                          'got %r' % y)
 
+    sparseseries = (y.__class__.__name__ == 'SparseSeries')
```

Comment:
Just testing the name of the class is a bit dodgy, I think it would be better to use an isinstance.